### PR TITLE
Postgres 11

### PR DESCRIPTION
**Pull request by jframe**
Test web3signer using Postgresql 11 as we want this to be recommended version for further db changes and postgres 10.6 crashed on OSX Monterey when running the tests.

* Changes the embedded postgresql library from opentable library to zonky as this let's us specify which postgresql version we want to use
* Update the version of postgresql we are testing against to the latest postgres 11 release

### CODE REVIEW

File: slashing-protection/src/testFixtures/java/db/DatabaseUtil.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      flyway.migrate();
+
+      final String databaseUrl =
+          String.format("jdbc:postgresql://localhost:%d/postgres", db.getPort());
+      final Jdbi jdbi =
+          DbConnection.createConnection(
+              databaseUrl, DatabaseUtil.USERNAME, DatabaseUtil.PASSWORD, null);
+      return new TestDatabaseInfo(db, jdbi);
+    } catch (IOException e) {
+      throw new RuntimeException("Unable to create embedded postgres database", e);
```
Comment (usmansaleem) [inline slashing-protection/src/testFixtures/java/db/DatabaseUtil.java:44]: nit: `UncheckedIOException` might be a better choice to wrap `IOException`.

**Review by usmansaleem — APPROVED** [review]
LGTM, minor nit-pick.

**Review by usmansaleem — APPROVED** [review]
LGTM

File: slashing-protection/src/testFixtures/java/db/DatabaseUtil.java (hunk 1 above)
Comment (jframe) [reply slashing-protection/src/testFixtures/java/db/DatabaseUtil.java]: done

**Review by usmansaleem — APPROVED** [review]
LGTM

**Pull request merged**